### ISSUE TITLE
Custom Exchange Name

### ISSUE THREAD

**Issue by TimGraf** [nameko/nameko#508]
Is it possible to use a custom exchange name?  I've tried using a different exchange name in the YAML configuration file other than `nameko-rpc`, say something like `foo-rpc`.  Once I change this RPC does not seem to work.

**Comment by TimGraf** [comment]
We found our issue.

**Comment by mattbennett** [comment]
@TimGraf can I ask what the motivation for changing the exchange name was?

I was actually considering dropping that feature because it didn't feel particularly useful.

**Comment by TimGraf** [comment]
We have a clustered RabbitMQ in our development environment hosted in AWS.  We originally had our RabbitMQ cluster behind an ELB which caused problems with RabbitMQ as it turns out.   

When we developed locally using Docker containers we didn’t see any issues.  As part of investigating our issues we had hoped to point our local services to our development environment RabbitMQ cluster and isolate our local services’ RPC calls by using a different exchange name.  

We are using the Django REST Framework for our REST API as gateway services to our internal Nameko RPC/PubSub services.

Our issue turned out to be the configuration on the DRF side which was missing the exchange name setting in the config so it was defaulting to “nameko-rpc”.   

In any event we resolved the ELB and clustered RabbitMQ issue as well as adding the exchange name to the config on the DRF side so we probably will not need to change the exchange name again.
